app/src/main/res/layout-land/admin_pin_activity.xml

```diff
@@ -104,7 +105,8 @@
           android:imeOptions="actionDone"
           android:inputType="numberPassword"
           android:maxLength="5"
-          android:text="@{viewModel.savedConfirmPin}" />
+          android:text="@{viewModel.savedConfirmPin}"
+          android:textCursorDrawable="@drawable/color_cursor" />
```

Comment:
It seems similar for all views (TextInputEditText)
can you add the textCursorDrawable to styles instead

Reply:
Sure thing